drivers/gpio/Kconfig.aw9523b

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+config GPIO_AW9523B
+	bool "AW9523B GPIO driver"
+	default y
+	depends on DT_HAS_AWINIC_AW9523B_GPIO_ENABLED
+	select I2C
+	select MFD
+	help
+	  Enable the AW9523B GPIO controller.
+
+config GPIO_AW9523B_INIT_PRIORITY
+	int "Init priority"
+	default 80
+	depends on GPIO_AW9523B
+	help
+	  AW9523B GPIO driver initialization priority.
```

Comment:
can you try and ditch this and just use `CONFIG_MFD_INIT_PRIORITY`? Both here and on the mfd driver. Then subpriorities will control sequencing correctly and we have two less priority options kicking around.

Reply:
Thank you for your suggestion. This is the ideal solution.